### ISSUE TITLE
Train on RTX A4000

### ISSUE THREAD

**Issue by Beastmaster** [vzyrianov/lidargen#9]
Thanks for sharing your code! While I have some questions.
How did you manage to train the model on A4000(I suppose 16G)? The bach size on "/workspace/third/lidargen/kitti.yml" is 24. I can only set to batch=4 on 3090GPU (24G). Did I miss something?

BTW, I trained kitti with batch=4, but got unstatisfying results.

**Comment by HaiyunW** [comment]
@Beastmaster Hello, I trained kitti-360 with bachsize=4 and got unstatisfying results, too.  Do you reslove this problem? I want to retry with bigger bachsize, but I‘m not sure if it will works.

**Comment by Beastmaster** [comment]
No. I didn't fix it. I compared pretrained model with my model, I got similar results(visually). I forgot exact numbers of evaluation metric.

**Comment by abhigoku10** [comment]
@Beastmaster @HaiyunW  how did u guys trained the model , is there any script available can you please let em know or share the script
